core/src/main/java/com/orientechnologies/orient/core/metadata/sequence/OSequence.java

```diff
@@ -159,8 +223,27 @@ public synchronized boolean updateParams(CreateParams params) {
       any = true;
     }
 
+    if (params.limitValue != null && this.getLimitValue() != params.limitValue) {
+      this.setLimitValue(params.limitValue);
+      any = true;
+    }
+
+    if (params.orderType != null && this.getOrderType() != params.orderType) {
+      this.setOrderType(params.orderType);
+      any = true;
+    }
+
+    if (params.recyclable != null && this.getRecyclable() != params.recyclable) {
+      this.setRecyclable(params.recyclable);
+      any = true;
+    }
+
+    if (params.turnLimitOff != null && params.turnLimitOff == true) {
+      this.setLimitValue(null);
+    }
+
     save();
-    reset();
+//    reset();
```

Comment:
I do not know the behaviour of reset, but if you can keep it out just remove the line of code

Reply:
Done, thanks